good-enough-practices-for-scientific-computing.tex

```diff
@@ -846,28 +846,30 @@ \subsection*{Version Control Systems}
 
 \subsection*{What Not to Put Under Version Control}
 
-Despite the benefits of version control systems, some types of files
-may \emph{not} be appropriate for version control.
+The benefits of version control systems don't apply equally to all file types.
```

Comment:
Overview: This section seemed to imply that you **can't** version control certain files. I reworded to make it more clear that you **can** but that you don't necessarily reap all the usual benefits.